src/Administration/Resources/app/administration/src/module/sw-plugin/view/sw-plugin-list/sw-plugin-list.html.twig

```diff
@@ -84,7 +85,7 @@
                                 {% endblock %}
 
                                 <template v-if="item.customFields">
-                                    {% block sw_plugin_list_grid_columns_actions_settings %}
```

Comment:
You rename the block. This could lead to plugin breaks.

Reply:
Absolutely right. I reverted this change